### PR TITLE
Update Juno documentation

### PR DESCRIPTION
**Pull request by jenswi-linaro**
Updates Juno documentation and removes obsolete scripts and patches.

Signed-off-by: Jens Wiklander <jens.wiklander@linaro.org>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -449,6 +334,7 @@ $ repo sync
 | FVP | `fvp.xml` | `fvp_stable.xml` |
 | HiKey | `hikey.xml` | `hikey_stable.xml` |
 | MediaTek MT8173 EVB Board | `mt8173-evb.xml` | `mt8173-evb_stable.xml` |
+| ARM Juno board| `juno.xml` | `n/a` |
```
Comment (pascal-brand-st-dev) [inline README.md]: Can you add the stable version as it will exist in few days?

**Comment by pascal-brand-st-dev** [comment]
`Reviewed-by: Pascal Brand <pascal.brand@linaro.org>`

@cedric-chaumont-st-dev Could you review it?

**Comment by cedric-chaumont-st** [comment]
@jenswi-linaro : 
- could you add the link to "Using Linaro's deliverable on Juno" page ?  
- could you add a note related to prebuilt-binaries and how to erase Juno's flash (if needed) ? 
- could you add a note related to GP testsuite potential issue (update of ramdisk.img) ? (maybe Ryan Harkin did the modification )

**Comment by jenswi-linaro** [comment]
I'll add the link and some references to how to flash it.
I'd like skip the GP-stuff for now as I didn't fully understand what you did to fix the problem.

**Comment by jenswi-linaro** [comment]
Update

**Comment by cedric-chaumont-st** [comment]
@jenswi-linaro : I can provide additional explanations or add a note that depending on the ramdisk.img configuration the GP testsuite cannot be supported.

**Comment by cedric-chaumont-st** [comment]
@jenswi-linaro : ok to skip the GP stuff,  I will add a note later (if you want)

**Comment by jenswi-linaro** [comment]
@cedric-chaumont-st Sounds good, perhaps you can make a later PR to add that. If you're OK with the current state, can you add your R-B please?

**Comment by cedric-chaumont-st** [comment]
Reviewed-by: Cedric Chaumont <cedric.chaumont@linaro.org>

**Comment by jenswi-linaro** [comment]
Tags applied

**Comment by jenswi-linaro** [comment]
Fixing the checkpatch errors.

**Pull request merged**